README.md

```diff
@@ -14,147 +14,38 @@ OpenFunction features but not limited to the following:
-| [`HEAD`](https://github.com/OpenFunction/OpenFunction/tree/main)          | &times;         | &radic;         | &radic;         | &radic;         |
+| OpenFunction                                                 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20+ |
+| ------------------------------------------------------------ | --------------- | --------------- | --------------- | ---------------- |
+| [`release-0.3`](https://github.com/OpenFunction/OpenFunction/tree/v0.3.0) | &radic;         | &radic;         | &radic;         | &radic;          |
+| [`release-0.4`](https://github.com/OpenFunction/OpenFunction/tree/v0.4.0) | &radic;         | &radic;         | &radic;         | &radic;          |
+| [`HEAD`](https://github.com/OpenFunction/OpenFunction/tree/main) | &times;         | &times;         | &radic;         | &radic;          |
 
 ## 🚀 QuickStart
 
-### Prerequisites
-
-The current version of OpenFunction requires that you have a Kubernetes cluster with version ``>=1.18.6``.
-
-In addition, you need to deploy several dependencies for the OpenFunction ```Builder``` and ```Serving```.
-
-You can use the following command to setup OpenFunction ```Builder``` and ```Serving```:
-
-```shell
-sh hack/deploy.sh --all
-```
-
-This command will install dependencies of all supported ```Builder``` and ```Serving``` to your cluster.
-
-You can also customize the installation with the following parameters:
-
-|Parameter|Comment|
-|---|---|
-| --all                              | Install all supported ```Builder``` and ```Serving``` |
-| --with-shipwright                  | Install Shipwright builder |
-| --with-knative                     | Install Knative serving runtime |
-| --with-openFuncAsync               | Install OpenFuncAsync serving runtime |
-| --with-ingress                     | Install nginx-ingress |
-| --region-cn                        | Use this if you have poor network connectivity to GitHub/Googleapis |
-
 ### Install OpenFunction
 
-You can install the OpenFunction platform by the following command:
+Now you can use `ofn`, the CLI of OpenFunction, to install and uninstall OpenFunction and its dependencies. 
 
-- Install the latest stable version
-
-```shell
-kubectl create -f https://github.com/OpenFunction/OpenFunction/releases/download/v0.4.0/bundle.yaml
-```
+Visit [ofn release (v0.5.0)](https://github.com/OpenFunction/cli/releases/tag/v0.5.0) to select the `ofn` cli to deploy to your cluster.
 
-- Install the development version
+After that you can install OpenFunction with one command:
 
-```shell
-kubectl create -f https://raw.githubusercontent.com/OpenFunction/OpenFunction/main/config/bundle.yaml
+```
+ofn install --all
 ```
 
-> Note: When using non-default namespaces, make sure that the ClusterRoleBinding in the namespace is adapted.
+See more at: [ofn install docs](https://github.com/OpenFunction/cli/blob/main/docs/install.md)
 
 ### Verify the installation
```

Comment:
Why need to verify the installation? Pod status check should be part of the installation.

Reply:
the `ofn` does check the readiness of the pods.

I'd remove this part.